test/conformance/CMakeLists.txt

```diff
@@ -94,6 +94,7 @@ add_subdirectory(event)
 add_subdirectory(queue)
 add_subdirectory(sampler)
 add_subdirectory(virtual_memory)
+add_subdirectory(exp_command_buffer)
```

Comment:
I'd move this down to around [118](https://github.com/oneapi-src/unified-runtime/pull/1089/commits/723c2eaf9fb9a11e8cd933bd922f84e8d016e54e#diff-1595fcdcdd0e6bf22312a2db5cbf98fb89265acbbc4bfa9887845dbe193a291fR118) with the other device code tests, maybe the problem is being caused by the `device_code` subdirectory being added after your test